SingleFile/main.cpp

```diff
@@ -927,7 +926,7 @@ VOID WINAPI Init (HMODULE mod) {
 	MH_DisableHook(NULL); // NULL = all hooks
 	MH_RemoveHook(NULL);
 	MH_Uninitialize();
-	FreeConsole();
+	FreeLibraryAndExitThread(mod, 0x1);

```

Comment:
@es3n1n Does this crash for you? I had this before it and it would always crash for me so I removed it, but it may just be me.

Reply:
almost forgot to push my fix for this, the problem was in hooked wndproc